DeveloperSetupGuide.md

```diff
@@ -263,13 +263,13 @@ following--you will need to change the values for user name, password, etc. base
 At this point, you should have credentials to access VSAC. If not, please refer to [Prerequisites](#prerequisites) for
 how to create these credentials and return here after you have confirmed you can access VSAC.
 
-Update your local copy of the file `REMS/.env` to reflect your credentials:
+Create a local copy of the file `REMS/.env` and rename that copy to `REMS/.env.local` to reflect your credentials:
```

Comment:
Should we add instructions in here about how to run the docker compose with the .env.local file?

Reply:
I'm unsure since I was able to do `docker-sync-stack start` with an .env.local containing the VSAC_API_KEY alongside the .env in the root of rems-setup, and I saw no issues with getting the CDS Hooks cards back after clicking Submit to REMS Admin.